src/views/Project/index.jsx

```diff
@@ -1,111 +1,102 @@
 import { hot } from 'react-hot-loader';
 import { Component, Fragment } from 'react';
 import Typography from '@material-ui/core/Typography';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import _ from 'lodash';
```

Comment:
Bringing in lodash carries a lot of weight. If you want the single function from it, install that instead and import it:

https://www.npmjs.com/package/lodash.uniqby

Reply:
ok